relay-dynamic-config/src/defaults.rs

```diff
@@ -539,7 +539,7 @@ fn span_metrics_reduced() -> impl IntoIterator<Item = MetricSpec> {
                     .always(),
                 Tag::with_key("span.domain")
                     .from_field("span.sentry_tags.domain")
-                    .when(is_db.clone()),
+                    .when(is_db.clone() | is_resource.clone()),
```

Comment:
It might not be necessary on `exclusive_time` (only on `exclusive_time_light`), but it will not increase cardinality because it's derived from description.